psl/psl-core/src/common/preview_features.rs

```diff
@@ -124,7 +124,7 @@ pub const ALL_PREVIEW_FEATURES: FeatureMap = FeatureMap {
         | UncheckedScalarInputs
     }),
     hidden: enumflags2::make_bitflags!(PreviewFeature::{
-        JsConnectors
+        DriverAdapters
```

Comment:
Note: We need to remove it from hidden "later"

Reply:
https://www.notion.so/prismaio/Node-Drivers-All-the-phases-0e9090072ae64f5b85b94ed9b36750b8?pvs=4#045d030ebfcb41ca9b75e488b74b2049